composer.json

```diff
@@ -0,0 +1,52 @@
+{
+    "name": "handcraftedinthealps/rest-routing-bundle",
+    "type": "symfony-bundle",
+    "description": "This bundle provides automatic route registration for the Controllers",
+    "keywords": ["rest", "routing"],
+    "license": "MIT",
+    "autoload": {
+        "psr-4": {
+            "HandcraftedInTheAlps\\RestRoutingBundle\\": ""
+        },
+        "files":[
+            "Controller/Annotations/NamePrefix.php",
+            "Controller/Annotations/NoRoute.php",
+            "Controller/Annotations/Prefix.php",
+            "Controller/Annotations/RouteResource.php",
+            "Controller/Annotations/Version.php",
+            "Routing/ClassResourceInterface.php"
+        ],
+        "exclude-from-classmap": [
+            "Resources/",
+            "Tests/"
+        ]
+    },
+    "require": {
+        "php": "^7.2",
+        "doctrine/inflector": "^1.0",
+        "friendsofsymfony/rest-bundle": "^2.8 || ^3.0",
```

Comment:
Does this really require the rest-bundle? Wouldn't it be possible to make this an optional dependency?

Reply:
Good catch can be a dev dependency just need to conflict not matching rest bundle versions.